Source/Fuse.Common/Internal/SystemFont.uno

```diff
@@ -11,6 +11,13 @@ namespace Fuse.Internal
 		public readonly int Index;
 		public readonly IEnumerable<string> Styles;
 
+		//iOS13+ - for default systemfont
+		internal bool IsCustomFont = true;
```

Comment:
I think the iOS specific fields can be `extern(IOS) internal bool IsCustomFont ...` if we also make the SetFontAttributes method `extern(IOS) internal void SetFontAttributes(...)`.

Reply:
I get this when using `extern(IOS) ...`:
```
fuselibs/Source/Fuse.Common/Internal/SystemFont.uno(69.4): E3102: There is nothing named IsCustomFont accessible in this scope. Are you missing a package reference?
```

Reply:
works for the SetFontAttributes method

Reply:
Line 69 is inside the SetFontAttributes method, so if you do `extern(IOS)` on both the method and these fields, it should build successfully.

Reply:
ok, sorted.